contrib/download-frozen-image-v2.sh

```diff
@@ -225,32 +256,57 @@ while [ $# -gt 0 ]; do
 					handle_single_manifest_v2 "$manifestJson"
 					;;
 				application/vnd.docker.distribution.manifest.list.v2+json)
+					Arch="$(go env GOARCH)"
 					layersFs="$(echo "$manifestJson" | jq --raw-output --compact-output '.manifests[]')"
 					IFS="$newlineIFS"
 					layers=( $layersFs )
 					unset IFS
 
 					found=""
 					# parse first level multi-arch manifest
-					for i in "${!layers[@]}"; do
-						layerMeta="${layers[$i]}"
-						maniArch="$(echo "$layerMeta" | jq --raw-output '.platform.architecture')"
-						if [ "$maniArch" = "$(go env GOARCH)" ]; then
-							digest="$(echo "$layerMeta" | jq --raw-output '.digest')"
-							# get second level single manifest
-							submanifestJson="$(
-								curl -fsSL \
-									-H "Authorization: Bearer $token" \
-									-H 'Accept: application/vnd.docker.distribution.manifest.v2+json' \
-									-H 'Accept: application/vnd.docker.distribution.manifest.list.v2+json' \
-									-H 'Accept: application/vnd.docker.distribution.manifest.v1+json' \
-									"$registryBase/v2/$image/manifests/$digest"
-							)"
-							handle_single_manifest_v2 "$submanifestJson"
-							found="found"
-							break
-						fi
-					done
+					if [ "${Arch}" = "arm64" ] || [ "${Arch}" = "arm" ]; then
+                                                fetch_arm_variant
```

Comment:
Looks like this line uses spaces, not tabs, which makes @tianon cry :wink:

Reply:
sorry~Fix it now😊

Reply:
🤗